frontend/pages/admin/IntegrationsPage/cards/AutomaticEnrollment/WindowsAutomaticEnrollmentPage/WindowsAutomaticEnrollmentPage.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useContext } from "react";
+
+import PATHS from "router/paths";
+import { AppContext } from "context/app";
+
+// @ts-ignore
+import InputField from "components/forms/fields/InputField";
+import BackLink from "components/BackLink";
+import MainContent from "components/MainContent";
+import CustomLink from "components/CustomLink/CustomLink";
+
+const generateMdmTermsOfUseUrl = (domain: string) => {
+  return `${domain}/api/v1/fleet/mdm/microsoft/terms_of_use`;
+};
+
+const generateMdmDiscoveryUrl = (domain: string) => {
+  return `${domain}/api/v1/fleet/mdm/microsoft/discovery`;
+};
+
+const baseClass = "windows-automatic-enrollment-page";
+
+const WindowsAutomaticEnrollmentPage = () => {
+  const { config } = useContext(AppContext);
+
+  return (
+    <MainContent className={baseClass}>
+      <>
+        <BackLink
+          text="Back to automatic enrollment"
+          path={PATHS.ADMIN_INTEGRATIONS_AUTOMATIC_ENROLLMENT}
+          className={`${baseClass}__back-to-automatic-enrollment`}
+        />
+        <h1>Azure Active Directory</h1>
+        <p>
+          The end user will see Microsoft&apos;s default initial setup. You can
+          further simplify the initial device setup with Autopilot (Apple DEP
+          equivalent).{" "}
```

Comment:
@noahtalerman, the parenthetical doesn't read quite right to me. Perhaps we could rephrase along the lines of:
"You can further simplify the initial device setup with Autopilot, which is similar to Apple's Device Enrollment Program (DEP)."

Reply:
@gillespi314 hmm, I see what you mean. I like your phrase Sarah. I would make one small tweak to call out Apple's new name for DEP: Automated Device Enrollment.

This way users used to the new name and old name can understand. We do this in other places in the UI.

So here's my proposed update: 

```
You can further simplify the initial device setup with Autopilot, which is similar to Apple's Automated Device Enrollment (DEP)
```

@marko-lisica what do you think?

Reply:
@noahtalerman @gillespi314 Makes sense to me.

Reply:
updated

Reply:
Nice. Hey @marko-lisica I made this update in [Figma](https://www.figma.com/file/BCoMp3gHJe8VyKwkYWKfSX/%2311766-Instructions-for-Autopilot-enrollment?type=design&node-id=2-130&mode=design) so the expected copy is reflected there.